jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy

```diff
@@ -242,7 +242,11 @@ def run_tls_source_it_test(String case_name) {
                         throw e
                     }
                 }
-                stash includes: 'go/src/github.com/pingcap/dm/cov_dir/**', name: "integration-cov-${case_name}"
+                try {
+                    stash includes: 'go/src/github.com/pingcap/dm/cov_dir/**', name: "integration-cov-${case_name}"
```

Comment:
this is to handle no test case in "run_single_it_test('others_3')" for now, so no file to stash